src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,47 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+export interface Movie {
+  title: string;
+  description?: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+}
```

Comment:
If you use this interface in several components - move it to separate file with types.